### ISSUE TITLE
Updater is blinking black or not showing correctly

### ISSUE THREAD

**Issue by hasezoey** [ValveSoftware/steam-for-linux#8965]
#### Your system information

* Steam client version (build number or date): nov 30 2022, at 17:04:29 (and previous versions since around ~october)
* Distribution (e.g. Ubuntu): Manjaro 22.0.0 (`/etc/lsb-release`)
* Opted into Steam client beta?: yes
* Have you checked for system updates?: yes

#### Please describe your issue in as much detail as possible:

Since around the fix for #8700 / #6451, the updater (the other parts than the changelog) is either blinking black, being black, or (as seen in the added video) not showing at all.

#### Steps for reproducing this issue:

Pre conditions: wait for a update on steam, and wait until steam automatically detects it and opens a window **in the background, minimized**
- Open the window from the minimized state
- Observe window
- Hover over a button (like `RESTART STEAM` or `CLOSE`)
- Observe that the window fully shows again (without blinking)

This does **not** happen when done manually through `Check for updates` or the updater starts in the foreground.

[Video showcasing the problem (in the "not showing at all" state)](https://www.youtube.com/watch?v=agtSj5oSbL8)

**Comment by kisak-valve** [comment]
Hello @hasezoey, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by hasezoey** [comment]
> Hello @hasezoey, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

right, forgot about that, available at https://gist.github.com/hasezoey/38d44d00810ed329ad793700f1343b82

PS: also happens on other systems (with manjaro)
